pom.xml

```diff
@@ -121,7 +121,7 @@
     <nuxeo.webengine.version>8.3-SNAPSHOT</nuxeo.webengine.version>
     <nuxeo.features.version>8.3-SNAPSHOT</nuxeo.features.version>
 
-    <nuxeo.connect.client.version>1.4.24</nuxeo.connect.client.version>
+    <nuxeo.connect.client.version>1.4.25-SNAPSHOT</nuxeo.connect.client.version>
```

Comment:
I still not understand why a snapshot branch is not allowing snapshot dependencies... but ok. It's just a pain. 😓